index.html

```diff
@@ -3443,9 +3648,36 @@ <h3 id="Damage">Damage</h3>
             the attack, you add DEX (once) to the damage.
          </p>
          <p>
-            If a level 1 character had +3 to INT and was making an INT based attack that
-            dealt Light physical damage: They would first make an attack with INT+INT bonus
-            (2d12+3+3). If successful they would then deal Light+INT physical damage (2+3).
+            If a level 1 character had +2 to INT and was making an INT based attack that
+            dealt Light physical damage (with no legendary ability upgrades): They would first make an attack with INT+INT bonus
+            (2d12+2+2). If successful they would then deal Light+INT physical damage (3+2).
+         </p>
+         <h4 id="Critical Damage">Critical Damage</h4>
+         <p>
+            Critical damage is equal to the total attack damage plus 50% (half the damage, round down, and add it to the total
+            damage). 
+         </p>
+         <p>
+            Critically hitting a target will result in the target suffering critical damage 
+            (there are some abilities that will also deal critical damage). 
+            Additional damage from abilities is also not multiplied on a critical
+            hit - this will always be described as "additional" damage. Critical damage does not stack and 
+            cannot be multiplied more than once. If damage is both glancing and critical, it is considered as normal 
+            damage and is not multiplied.
+         </p>
+         <h4 id="Glancing Damage">Glancing Damage</h4>
+         <p>
+            Glancing damage is equal to half the total attack damage. 
+         </p>
+         <p>
+            Glancing damage can occur when you make an attack as part of an ability and roll lower than the AC but 
+            do not critically miss. The attack is still a hit but will deal half attack damage. Additional damage is 
+            again not affected by glancing and is applied as normal.
+         </p>
+         <p>
+            Other abilities and conditions can also cause glancing damage, but glancing damage does not stack and cannot be 
```

Comment:
I'd say glancing would always take precedent - feels weird that something can be both a crit and essentially miss